src/main/scala/it/pagopa/interop/apigateway/api/impl/GatewayApiServiceImpl.scala

```diff
@@ -505,7 +505,7 @@ final case class GatewayApiServiceImpl(
       latestDescriptor <- eService.latestAvailableDescriptor
       state            <- latestDescriptor.state.toModel.toFuture
       allAttributesIds = latestDescriptor.attributes.allIds
-      attributes <- attributeRegistryProcessService.getBulkAttributes(allAttributesIds)
+      attributes <- attributeRegistryProcessService.loadBulkAttributes(allAttributesIds)
```

Comment:
```suggestion
      attributes <- attributeRegistryProcessService.getBulkAttributes(allAttributesIds)
```
Note: load is the logic that creates new attributes starting from IPA